applovin_max/ios/Classes/AppLovinMAX.m

```diff
@@ -422,24 +404,7 @@ - (void)setTargetingDataMaximumAdContentRating:(nullable NSNumber *)maximumAdCon
         return;
     }
     
-    ALAdContentRating rating = ALAdContentRatingNone;
-    
-    int intVal = maximumAdContentRating ? maximumAdContentRating.intValue : 0;
-    
-    if ( intVal == 1 )
-    {
-        rating = ALAdContentRatingAllAudiences;
-    }
-    else if ( intVal == 2 )
-    {
-        rating = ALAdContentRatingEveryoneOverTwelve;
-    }
-    else if ( intVal == 3 )
-    {
-        rating = ALAdContentRatingMatureAudiences;
-    }
-    
-    self.sdk.targetingData.maximumAdContentRating = rating;
+    self.sdk.targetingData.maximumAdContentRating = [[self class] getAppLovinAdContentRating: maximumAdContentRating];
```

Comment:
To avoid `[self class]` you can make the getter a non-static/class method.

Reply:
Updated to be a non-static method.